app/forms/declaration/TransportDetails.scala

```diff
@@ -38,15 +38,17 @@ object TransportDetails {
 
   implicit val formats = Json.format[TransportDetails]
 
-  val formMapping = mapping(
+  val formMapping: Mapping[TransportDetails] = mapping(
     "meansOfTransportCrossingTheBorderNationality" -> optional(
       text()
         .verifying(
           "supplementary.transportInfo.meansOfTransport.crossingTheBorder.nationality.error.incorrect",
           isContainedIn(allCountries.map(_.countryName))
         )
     ),
-    "container" -> boolean,
```

Comment:
The `boolean` mapping defaults to false if it is absent. The workaround below ensures it is populated by the user